src/Components/TheoryPage.js

```diff
@@ -0,0 +1,989 @@
+                    setTheorySubject('Vertical Stack');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Vertical Stack</Text>
+                </TouchableHighlight>
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(1);
+                    setTheorySubject('Horizontal Stack');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Horizontal Stack</Text>
+                </TouchableHighlight>
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(2);
+                    setTheorySubject('Endzone Offense');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Endzone Offense</Text>
+                </TouchableHighlight>
+
+                <TouchableHighlight
+                  style={styles.subjectButton}
+                  onPress={() => {
+                    setSelectedIndex(3);
+                    setTheorySubject('Handling');
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.subjectText}>Handling</Text>
+                </TouchableHighlight>
+
+                <TouchableHighlight
+                  style={styles.returnButton}
+                  onPress={() => {
+                    setModalVisible(!modalVisible);
+                  }}
+                >
+                  <Text style={styles.textStyle}>Return</Text>
+                </TouchableHighlight>
+              </View>
+            </View>
+          </Modal>
+        </View>
+        <View style={styles.displayTheory}>
+          <TouchableHighlight
+            style={styles.subjectModal}
+            onPress={() => {
+              setModalVisible(true);
+            }}
+          >
+            <View>
+              <View style={styles.dropdown}>
+                <Text style={{ ...styles.textStyle, color: theme.COLOR_PRIMARY }}>{theorySubject}</Text>
+                <MaterialCommunityIcons name="chevron-down" color={theme.COLOR_PRIMARY} size={26} />
+              </View>
+              <View style={styles.lines} />
+            </View>
+          </TouchableHighlight>
+        </View>
+        <View style={styles.displayTheory}>
+          <View>{displayTheory()}</View>
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default function TheoryPage() {
+  return (
+    <Tab.Navigator
+      initialRouteName="Dictionary"
+      activeColor={theme.COLOR_PRIMARY}
+      inactiveColor={theme.COLOR_SECONDARY}
+      barStyle={{ backgroundColor: theme.BACKGROUND_COLOR }}
+      shifting
```

Comment:
It makes it a bit more unclear, since the user does not understand what the tabs are for without clicking them

Reply:
yes we will keep the title. I want to keep the tab to navigate